Ray.jl/test/runtime.jl

```diff
@@ -0,0 +1,39 @@
+@testest "control logs_dir" begin
+    @testest "defaul logs_dir" begin
+        # we're running this after `init` has been called, so logs should exist already:
+        logs = readdir("/tmp/ray/session_latest/logs"; join=true)
+        @test any(contains("julia-core-driver"), logs)
+    end
+
+    @testset "custom logs_dir" begin
+        mktempdir() do logs_dir
+            cp("/tmp/ray/session_latest/logs/raylet.out", joinpath(logs_dir, "raylet.out"))
```

Comment:
why do we need to copy the `raylet.out` over?

Reply:
Why not use a symlink for this purpose? Seems clearer

Reply:
eh, it's throwaway in a tempdir for a test.  these files _can_ get big when they're running for a long time but i don't think that's likely to have a meaningful impact in this test suite

Reply:
just gonna leave it as cp for now to keep this moving.